packages/rich-text/src/insert-line-separator.js

```diff
@@ -11,9 +11,9 @@ import { LINE_SEPARATOR } from './special-characters';
  * `startIndex`. Any content between `startIndex` and `endIndex` will be
  * removed. Indices are retrieved from the selection if none are provided.
  *
- * @param {Object} value         Value to modify.
- * @param {number} startIndex    Start index.
- * @param {number} endIndex      End index.
+ * @param {Object} value        Value to modify.
```

Comment:
Slight inconsistency here, compared to https://github.com/WordPress/gutenberg/pull/14220/files#diff-9ab2f40d2dac2f7335282d5df03a013fR18.

Reply:
ok, fixed spacing